build/build.py

```diff
@@ -256,6 +260,9 @@ def write_bazelrc(python_bin_path=None, remote_build=None,
     if rocm_toolkit_path:
       f.write("build --action_env ROCM_PATH=\"{rocm_toolkit_path}\"\n"
               .format(rocm_toolkit_path=rocm_toolkit_path))
+    if rocm_amdgpu_targets:
+      f.write("build --action_env TF_ROCM_AMDGPU_TARGETS=\"{rocm_targets}\"\n"
```

Comment:
Same comment. Just use `--config=rocm`.

Is this because you may need different targets that are not in the static .bazelrc flie?

Reply:
The main reason I added this was because the ```parser.add_argument("--rocm_amdgpu_targets", .... ``` were still there, I am ok to drop this from the build.py and keep it in the static .bazelrc, but we should also update the arguments list of scripts as well (same for cuda_compute_capabilities).
Please let me know your decision to update the PR.

Reply:
Yeah, let's do it. I would prefer to keep that in the static .bazelrc.